modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/visualization_modes/VisualizationModes.tsx

```diff
@@ -46,6 +22,8 @@ const VISUALIZATION_MODE_COMPONENT_MAP: {
 	table: Table,
 };
 
+const ORDERED_DEFAULT_VISUALIZATION_MODES = [...DEFAULT_VISUALIZATION_MODES].reverse();
```

Comment:
@markocikos :pray: 
After moving things to `/utils/constants` I've realized that we have different orders when it comes to show Visualizaton Modes (DSM tabs: Table, List, Cards) and DSM Settings dropdown / FDS views dropdown (Cards, List, Table).
If we decide to use the same order everywhere, we will need to change also how to return them in the fragment.